handout/testing/components/injecting-dependencies.md

```diff
@@ -46,39 +46,52 @@ class MockQuoteService {
   }
 }
 
-import {QuoteService} from './quote.service.ts';
-import {QuoteComponent} from './quote.component.ts';
+import {QuoteService} from 'quote.service.ts';
+import {QuoteComponent} from 'quote.component.ts';
+import {provide} from '@angular/core';
 import {
+  async,
   expect,
   it,
   describe,
-  async
-  TestComponentBuilder,
-  beforeEachProvider
+  inject,
+  beforeEach,
+  beforeEachProvider,
 } from '@angular/core/testing';
 
+import {
+  TestComponentBuilder,
+  ComponentFixture,
+}
+from '@angular/compiler/testing';
+
 describe('Testing Quote Component', () => {
-  beforeEachProviders(() => {
-    provide(QuoteService: {useClass: MockQuoteService})
-  });
 
-  it('Should get quote', async(inject([TestComponentBuilder], (tcb) => {
-    return tcb.createAsync(QuoteComponent).then(fixture) => {
```

Comment:
`injectAsync` might have required the promise to be returned to know when the async action has completed, but now that we have `async` Angular uses the test zone to make sure all async operations have completed (queues are empty) before wrapping up.  For this reason, we don't have to return.